### PR TITLE
[HOTFIX] Connection gating logs

### PR DESCRIPTION
**Pull request by nholland94**

### CODE REVIEW

File: src/app/libp2p_helper/src/codanet.go [hunk 1]
```diff
@@ -171,6 +184,12 @@ func (gs *CodaGatingState) InterceptPeerDial(p peer.ID) (allow bool) {
 // resolved the peer's addrs, and prior to dialling each.
 func (gs *CodaGatingState) InterceptAddrDial(id peer.ID, addr ma.Multiaddr) (allow bool) {
 	allow = (!gs.DeniedPeers.Contains(id) || gs.AllowedPeers.Contains(id)) && !gs.AddrFilters.AddrBlocked(addr)
+
+	if !allow {
+		gs.logger.Infof("disallowing peer dial from: %v", p)
```
Comment (mrmr1993) [inline src/app/libp2p_helper/src/codanet.go:189]: Isn't the equivalent of this logged [here](https://github.com/libp2p/go-libp2p-swarm/blob/830b5b61da7fee5e6ead8b840399f1895e1aa3d3/swarm_dial.go#L228) already?

**Review by mrmr1993 — APPROVED** [review]

File: src/app/libp2p_helper/src/codanet.go [hunk 2]
```diff
@@ -129,13 +129,16 @@ type customValidator struct {
 // https://godoc.org/github.com/libp2p/go-libp2p-core/connmgr#ConnectionGating
 // the comments of the functions below are taken from those docs.
 type CodaGatingState struct {
+	logger       *logging.Logger
```
Comment (mrmr1993) [inline src/app/libp2p_helper/src/codanet.go:132]: Should this be `logging.EventLogger` instead?

File: src/app/libp2p_helper/src/codanet.go (hunk 1 above)
Comment (nholland94) [reply src/app/libp2p_helper/src/codanet.go:189]: These are different layers the connection gating goes through. If we reject the first one, we won't get here.

File: src/app/libp2p_helper/src/codanet.go (hunk 2 above)
Comment (nholland94) [reply src/app/libp2p_helper/src/codanet.go:132]: Wasn't using events, so I figured it didn't matter. Can change if you think it's better to leave the interface more general.

**Pull request merged**